### PR TITLE
Add type hints to blueman.gui

### PR DESCRIPTION
**Pull request by cschramm**

### CODE REVIEW

File: blueman/gui/GenericList.py [hunk 1]
```diff
@@ -127,11 +141,11 @@ def get(self, iterid, *items):
...
                         ret[items[i]] = self.liststore.get(tree_iter, self.ids[items[i]])[0]
         else:
-            return False
+            return {}
 
         return ret
 
-    def get_iter(self, path):
+    def get_iter(self, path: Union[Gtk.TreePath, int, str]) -> Optional[Gtk.TreeIter]:
         if path is None:
```
Comment (infirit) [inline blueman/gui/GenericList.py:149]: Path may be None so the Union is optional, right? If not then the check for None can be removed.

File: blueman/gui/GtkAnimation.py [hunk 2]
```diff
@@ -7,6 +7,8 @@
 
 gi.require_version("Gtk", "3.0")
 from gi.repository import Gtk
+gi.require_version("Gdk", "3.0")
```
Comment (infirit) [inline blueman/gui/GtkAnimation.py:10]: Is there a warning on the Gdk version?

File: blueman/gui/GtkAnimation.py [hunk 3]
```diff
@@ -138,12 +142,14 @@ def on_draw(self, widget, cr):
...
-    def state_changed(self, state):
+        return False
+
+    def state_changed(self, state: float) -> None:
         self.tw.queue_draw()
 
 
 class CellFade(AnimBase):
-    def __init__(self, tw, path, columns=None):
+    def __init__(self, tw: Gtk.TreeView, path: Gtk.TreePath, columns: Iterable[int]) -> None:
```
Comment (infirit) [inline blueman/gui/GtkAnimation.py:152]: tw is not a plain TreeView but ManagerDeviceList.

File: blueman/gui/GtkAnimation.py (hunk 2 above)
Comment (cschramm) [reply blueman/gui/GtkAnimation.py:10]: Yup, 
> PyGIWarning: Gdk was imported without specifying a version first. Use gi.require_version('Gdk', '3.0') before import to ensure that the right version gets loaded.

File: blueman/gui/GtkAnimation.py (hunk 3 above)
Comment (cschramm) [reply blueman/gui/GtkAnimation.py:152]: Does it have to be? From the attributes `CellFade` expects I think any `Gtk.TreeView` should work.

Comment (infirit) [reply blueman/gui/GtkAnimation.py:152]: I think when the pygobject stubs are installed mypy will complain. But we can cross that bridge when that happens.

File: blueman/gui/GenericList.py (hunk 1 above)
Comment (cschramm) [reply blueman/gui/GenericList.py:149]: You're right. At least `ManagerDeviceList.level_setup_event` might theoretically pass `None`, however `mypy` does not know that due to no GObject stubs as it sees just `Any`. Also `mypy --strict-equality` would catch this but it sees `path` as `Any` as well, as `Gtk.TreePath` is `Any`.

**Review by infirit — APPROVED** [review]

**Pull request merged**